R/UK.R

```diff
@@ -0,0 +1,519 @@
+#' United Kingdom Class for downloading, cleaning and processing notification
+#' data.
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for an example Country.
+#'
+#' @details Inherits from `DataClass`
+#' @source https://coronavirus.data.gov.uk/details/download #nolint
+#' @examples
+#' \dontrun{
+#' region <- Italy$new(verbose = TRUE, steps = TRUE)
+#' region$download()
+#' region$clean()
+#' region$process()
```

Comment:
please document functionality here to make reviewing easier.

Reply:
(can you add some examples using the different custom UK features as well please)